apps/aecore/src/aec_vm_chain.erl

```diff
@@ -499,6 +500,13 @@ oracle_typerep(?ABI_AEVM_SOPHIA_1, BinaryFormat) ->
     end.
 
 %%    Address
+addr_is_payable(Addr, #state{ trees = CTrees }) ->
+    Trees = get_on_chain_trees(CTrees),
+    case aec_accounts_trees:lookup(Addr, aec_trees:accounts(Trees)) of
+        none             -> {ok, bool2word(false)};
```

Comment:
Is this right? If an address does not exist shouldn't it count as payable? Surely a spend to it will succeed (and create a payable account in the process)?